src/netbase.cpp

```diff
@@ -559,12 +559,8 @@ std::function<std::unique_ptr<Sock>(int, int, int)> CreateSock = CreateSockOS;
 template<typename... Args>
 static void LogConnectFailure(bool manual_connection, util::ConstevalFormatString<sizeof...(Args)> fmt, const Args&... args)
 {
-    std::string error_message = tfm::format(fmt, args...);
-    if (manual_connection) {
-        LogPrintf("%s\n", error_message);
-    } else {
-        LogDebug(BCLog::NET, "%s\n", error_message);
-    }
+    auto log_level{manual_connection ? BCLog::Level::Info : BCLog::Level::Debug};
+    LogPrintLevel(BCLog::NET, log_level, fmt, args...);
```

Comment:
Is the `LogFlags::NONE` -> `BCLog::NET` category in case of `manual_connection` a behavior change?